main.py

```diff
@@ -0,0 +1,116 @@
+#Amazon data set VADER analysis
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import nltk
+from nltk.sentiment import SentimentIntensityAnalyzer
+from tqdm.notebook import tqdm
+
+#nltk.download('punkt')
+#nltk.download('averaged_perceptron_tagger')
+#nltk.download('maxent_ne_chunker')
+#nltk.download('words')
+#nltk.download('vader_lexicon')
+
+plt.style.use('ggplot')#
+
+
+#reading data and basic analysis
+data = pd.read_csv('./data/Reviews.csv')
+data = data.head(1000) #using 1000 rows
+#print(data.shape)
+
+ax = data['Score'].value_counts().sort_index().plot(kind = 'bar',title = 'Count of Reviews',\
+                                               figsize = (10, 5))
+ax.set_xlabel('Review Stars')
+plt.show()
+
+polarity = []
+sia = SentimentIntensityAnalyzer()
+
+for i in data['Id']:
+    tmp = []
+    tmp.append(i)
+    polarity.append(tmp)
+q = 0
+for i in data['Text']:
+    polarity[q].append(sia.polarity_scores(i))
+    q += 1

```

Comment:
Ok, now I get it you want to create a matrix with two columns, the first with an index and the second with the polarity score. Do it at the same time:

```suggestion
polarities = []
for index, row in data.iterrows():
   id = row['Id']
   polarity = sia.polarity_scores(row['Text'])
   polarities.append([index, polarity])
```
I recommend using ChatGPT for this, e.g. asking it to 
`Write a python code iterating over two columns of a data time frame` gives you good examples (obviously you need to verify the answers, because sometime they are wrong)


There is also some pandas magic that allows you to create the `sentence_scores` dataframe directly from `data`, but this is sufficient.

Reply:
fixed